zerver/actions/message_send.py

```diff
@@ -94,6 +94,7 @@
     query_for_ids,
 )
 from zerver.tornado.django_api import send_event
+from zerver.views.scheduled_messages import get_scheduled_message
```

Comment:
This is probably an import cycle; I'd probably put that function in a `zerver.lib.scheduled_messages` to avoid that.

Reply:
Done.